experiments/cvat_api_wrapper.py

```diff
@@ -29,13 +29,37 @@ def get_task_job(cvat_url_root, key, task_id):
     res = requests.get(f"{cvat_url_root}/api/v1/tasks/{task_id}/jobs", cookies=key)
     return res.json()[0]
 
+def get_username_id(cvat_url_root, key, username):
+    res = requests.get(f"{cvat_url_root}/api/v1/users", cookies=key)
+
+    user_id = next(u["id"] for u in res.json()["results"] if u["username"] == username)
+    return user_id
+
+def assign_task_to_user(cvat_url_root, key, task_id, username):
+    job_id = get_task_job(cvat_url_root, key, task_id)["id"]
+    user_id = get_username_id(cvat_url_root, key, username)
+
+    p = {"assignee_id": user_id}
+    h = {"X-CSRFToken": key["csrftoken"]}
+    res = requests.patch(f"{cvat_url_root}/api/v1/jobs/{job_id}", \
+                                            json=p, headers=h, cookies=key)
```

Comment:
nit: não precisa do `\` pra quebrar em multilinhas quando tem um `()`
```suggestion
    res = requests.patch(
        f"{cvat_url_root}/api/v1/jobs/{job_id}", json=p, headers=h, cookies=key)
```